src/main/resources/com/google/api/codegen/java/discovery_fragment.snip

```diff
@@ -66,11 +66,18 @@
         JsonFactory jsonFactory = JacksonFactory.getDefaultInstance();
 
         {@ApiName} {@serviceName()} = new {@ApiName}.Builder(httpTransport, jsonFactory, credential)
-          .setApplicationName("Google Cloud Platform Sample")
-          .build();
+            .setApplicationName("Google Cloud Platform Sample")
+            .build();
 
         {@initialize(method)}
         {@methodCall(method)}
+        @if context.hasMediaDownload(method)
+
+          // {@TODO()} Download desired media content. For more information, see:
```

Comment:
Right, so my comment is that the `Download media content if desired.` (either wording) does not address that use case. Is there a way to distinguish the two cases? If so, we should tailor the wording. If not, let's file an issue to follow up on this.